tests/PLAYWRIGHT_GUIDE.md

```
@@ -0,0 +1,467 @@
+    await fillForm(page, { name, type: "Department" });
+    await submitForm(page);
+    await expectToast(page, "Organization created successfully");
+  });
+});
+```
+
+## File Organization
+
+Place test files in the matching feature directory:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)